<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Эмуляция контроллера ELAM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="listPLC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>25</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Цифровые входы</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableView" name="tableInputs"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Хранимые регистры</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="tableHoldingRegs"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Входные регистры</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="tableInputRegs"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Телеуправление</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableView" name="tableCoils"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="console">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Remove"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Script"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Add"/>
   <addaction name="action_Remove"/>
   <addaction name="separator"/>
   <addaction name="action_Settings"/>
   <addaction name="separator"/>
   <addaction name="action_Script"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/edit_add.png</normaloff>:/icons/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить...</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/favicon_settings.png</normaloff>:/icons/favicon_settings.png</iconset>
   </property>
   <property name="text">
    <string>Настройки порта...</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/edit_remove.png</normaloff>:/icons/edit_remove.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Создать скрипт</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Сохранить скрипт...</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Открыть скрипт...</string>
   </property>
  </action>
  <action name="action_Script">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/accessories-text-editor.png</normaloff>:/icons/accessories-text-editor.png</iconset>
   </property>
   <property name="text">
    <string>Показать скрипт</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="elam_emulator.qrc">
     <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listPLC</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableInputs</tabstop>
  <tabstop>tableHoldingRegs</tabstop>
  <tabstop>tableInputRegs</tabstop>
  <tabstop>tableCoils</tabstop>
  <tabstop>console</tabstop>
 </tabstops>
 <resources>
  <include location="elam_emulator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listPLC</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changePLC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changePLC()</slot>
 </slots>
</ui>
